Docs/releases/release-3.5.0.md

```diff
@@ -34,4 +35,4 @@ You will need to update your Creator Companion to version 2.2.0 in order to mana
 * The Editor may crash when updating a shader graph reference by another shader using UsePass. This is an issue with Unity 2022.3.6f1 and is fixed in 2022.3.14f1.
 * Unity 2022 sometimes causes Rider's debugger to stop for unhandled exceptions in Unity's IMGUI.
 	* Please refer to [this workaround](https://forum.unity.com/threads/rider-debugger-breaks-on-unhandled-exception.1135879/#post-7305256) and [Jetbrains's bug tracker](https://youtrack.jetbrains.com/issue/RIDER-64944) for more information.
-* Spatialized Audio Sources can create warnings when entering playmode or adjusting their settings.
\ No newline at end of file
+* Spatialized Audio Sources can create warnings when entering playmode or adjusting their settings.
```

Comment:
I'm not seeing any changes in this line - it might be a fluke.

Reply:
I think this is because there is no EOL at the EOF. GitHub web editor will adds EOL at EOF is not exists.